### PR TITLE
Refactor and added the slider and inline list

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Hello everyone I have made a slider and also a Inline List also included it in readme.md
Please review it once Thanks a lot
Live UI-https://deploy-preview-10--piston-ui.netlify.app/html-folder/slider.html

### CODE REVIEW

File: html-folder/alerts.html [hunk 1]
```diff
@@ -85,7 +85,9 @@ <h2>Get Started</h2>
                                 <li class="aside-ul-li margin-bot-med">
                                     <a href="./ratings.html" class="aside-link link-btn ">Ratings</a>
                                 </li>
-
+                                <li class="aside-ul-li margin-bot-med">
+                                    <a href="./ratings.html" class="aside-link link-btn ">Slider</a>
```
Comment (vrx29) [inline html-folder/alerts.html:89]: please remove the extra space after last class name link-btn.

File: html-folder/alerts.html [hunk 2]
```diff
@@ -98,14 +100,14 @@ <h1 class="text-shadow"> PISTON <span class="highlight-orange-text">UI</span></h
...
+            <a href="https://github.com/Dev-AkshatGupta" class="nav-btn-icon text-black" target="blank"><i
                     class="fab fa-github"></i>
             </a>
-            <a href="https://twitter.com/Akshat86604233" class="nav-btn-icon highlight-white-text" target="blank"><i
+            <a href="https://twitter.com/Akshat86604233" class="nav-btn-icon text-black" target="blank"><i
                     class="fab fa-twitter"></i>
             </a>
             <a href="https://www.linkedin.com/in/akshat-gupta-257914157/" class="nav-btn-icon
-        highlight-white-text" target="blank"><i class="fab fa-linkedin-in"></i>
+    text-black" target="blank"><i class="fab fa-linkedin-in"></i>
```
Comment (vrx29) [inline html-folder/alerts.html:110]: target should be  _blank please change that

File: html-folder/alerts.html [hunk 3]
```diff
@@ -114,28 +116,20 @@ <h1 class="text-shadow"> PISTON <span class="highlight-orange-text">UI</span></h
     <div class="universal-container">
         <aside class="aside-nav box-shadow-right">
             <div class="aside-container-pri ">
-                <a href="" class="link-btn">
+                <a href="../index.html" class="link-btn">
                     <h2>Get Started</h2>
                 </a>
                 <ul class="aside-ul">
                     <li class="aside-ul-li margin-bot-med">
                         <a href="../index.html" class="aside-link link-btn ">Introduction</a>
```
Comment (vrx29) [inline html-folder/alerts.html:124]: remove space in class name

File: html-folder/alerts.html (hunk 2 above)
Comment (atul-gairola) [reply html-folder/alerts.html:110]: both _blank and blank work so no worries for that

**Review by atul-gairola — APPROVED** [review]
Seems good to go!

**Comment by Dev-AkshatGupta** [comment]
> Seems good to go!

Thanks for reviewing it

File: html-folder/alerts.html (hunk 3 above)
Comment (Dev-AkshatGupta) [reply html-folder/alerts.html:124]: Thanks for the review and your value able suggestions I will make the changes suggested by you

File: html-folder/alerts.html (hunk 1 above)
Comment (Dev-AkshatGupta) [reply html-folder/alerts.html:89]: Done with this

File: html-folder/alerts.html (hunk 2 above)
Comment (Dev-AkshatGupta) [reply html-folder/alerts.html:110]: We can go both ways I guess

**Pull request merged**